apps/dg/libraries/slickgrid/plugins/slick.rowselectionmodel.js

```diff
@@ -34,10 +34,12 @@
 
     function wrapHandler(handler) {
       return function () {
+        var result;
```

Comment:
I don't think there is any reason to remove it, more concerned that it get in the right version,  if it is depended on.